examples/jsm/postprocessing/UnrealBloomPass.js

```diff
@@ -130,6 +130,7 @@ class UnrealBloomPass extends Pass {
 			vertexShader: copyShader.vertexShader,
 			fragmentShader: copyShader.fragmentShader,
 			blending: AdditiveBlending,
```

Comment:
A so-called `materialCopy` that does additive blending is not copying. I think that is poorly-named.

Do you still think `premultipliedAlpha` is correct? I find trying to track the logic of the code in this file a bit daunting...

If you are not sure, I wonder if changing the behavior for users is worth it?

//

On a related note, I think the existing example has too much bloom effect, and looks better with this one-line change. IMO, there would be no need to change the parameters to try to match the previous look.

Reply:
> A so-called materialCopy that does additive blending is not copying. I think that is poorly-named.

Agreed. E.g. `materialBlend` sounds more appropriate.

> Do you still think premultipliedAlpha is correct?

Yes. The pass additively blends over the input. Considering how `CopyPass` is implemented, `premultipliedAlpha` should be set to `true`.

> If you are not sure, I wonder if changing the behavior for users is worth it?

Let me think about this. That is a valid point since even without the change the effect looks "right".

> On a related note, I think the existing example has too much bloom effect, and looks better with this one-line change. IMO, there would be no need to change the parameters to try to match the previous look.

Yeah, I think I got a little bit excited when `UnrealBloomPass` finally produces a correctly tone-mapped result in the right color space so I couldn't have enough bloom 😅 . I'm fine with turning it down a bit.

Reply:
>Considering how CopyPass is implemented, premultipliedAlpha should be set to true.

We should set `.premultipliedAlpha` to true only if the shader outputs premultiplied RGB values. So, the question is: does it?

Reply:
Yes, I think so. The opacity scales all color components, not just alpha.

Reply:
This effect was not written to properly accommodate the premultiplied output from RenderPass.

Given this PR causes breakage for users, and given the change here sill leaves inconsistencies in the code, I'd suggest doing nothing for now.

Reply:
> , and given the change here sill leaves inconsistencies in the code

On what inconsistencies are you referring to? Is there something else to fix?

Reply:
But I agree the breakage isn't worth the overall change. If someone repost a bug related to blending that can be fixed by setting the `premultipliedAlpha` to `true`, we can reconsider the PR.

Reply:
When implementing the fiddle for https://github.com/mrdoob/three.js/issues/24703#issuecomment-1584249800, I've noticed that the bloom is too strong given standard values like `1`. It seems enabling `premultipliedAlpha` as suggested in the PR produces a more consistent result.